### DISCUSSION TITLE
Open the subscription page on app launch

### DISCUSSION THREAD

**Discussion by Mike200109** [libre-tube/LibreTube#5809] [Ideas]
Add an option to select which tab to open on app launch. Now it always open home page even If I disable the home page. 
I want it to always open subscription tab on launch. Most of the videos are bad and unorganised in home page. 
I only use the subscription page and it is a pain to switch tab everytime after launch

**Comment by heyitschloehawthorne** [comment]
I'd love to have this feature as well, or being able to add subscriptions to the homepage at least.

**Comment by shuvashish76** [comment]
> I want it to always open subscription tab on launch.

@Mike200109 Settings > Appearance > Order > click on the small _Home icon_ next to Subscriptions.

> I'd love to have this feature as well, or being able to add subscriptions to the homepage at least.

@heyitschloehawthorne what you're asking is not same as the original post. You can add _Featured_ i.e. latest 20 videos from your subscription list to home from Settings > Appearance > Home tab content > turn on the Featured checkbox on.
You can't manage your subscriptions from there as Home is not intended for it. We've dedicated Subscription page on our navigation bar so makes no sense to add it to Home page.

**Reply by Mike200109** [reply]
Thanks I missed it.😅😅😅
